static/templyte/src/app/download-template/download-template.component.html

```diff
@@ -0,0 +1,39 @@
+<mat-toolbar class="mat-elevation-z3">
+    <nav class="navbar">
+        <ul class="nav navbar-right">
+            <li>
+                <button mat-button (click)="goHome()">Home</button>
+            </li>
+            <li>
+                <button mat-button (click)="createTemplate()">Create Template</button>
+            </li>
+            <li>
+                <button mat-button (click)="myTemplates()">My Templates</button>
+            </li>
+            <li>
+                <button mat-button (click)="logout()">Logout</button>
+            </li>
+        </ul>
+    </nav>
+</mat-toolbar>
+
+<mat-card class="card-download">
+    <h1>Download a Template</h1>
+
+    <h1>{{(templateDirectoryInfo | async)?.templateName}}</h1>
+    <!--TODO: Add description-->
+
+    <div>
+        <div *ngFor="let variable of (templateRenderInfo | async)?.variables">
+            <p>{{variable.name}}</p>
+            <p>{{variable.description}}</p>
```

Comment:
Styling for this and the page as a whole isn't implemented. Does @CaylaVinzons  still have time to finish styling before the end of the sprint? - if so, it should be in a different PR.